cognition_objects/conversation.py

```diff
@@ -25,6 +25,20 @@ def get(project_id: str, conversation_id: str) -> CognitionConversation:
     )
 
 
+def get_all_with_org() -> List[CognitionConversation]:
+    query = """
+    SELECT
+        cc.*,
+        cp.organization_id
+    FROM
+        cognition.conversation cc
+    JOIN
+        cognition.project cp
+    ON
+        cc.project_id = cp.id;"""
```

Comment:
doesn't match our common sql style:

```
SELECT cc.*, cp.organization_id
FROM cognition.conversation cc
INNER JOIN cognition.project cp
    ON cc.project_id = cp.id
```
Also is there a specific reason why this isn't an orm sql alchemy function?

Also the xxx_with name usually indicates a filter. maybe `get_all_including_org_id` or `get_all_extend_org_id`

Reply:
I tried it with sqlalchemy and always had to flatten the resulting org id with list comprehension